Engine/Annotations/Annotation.cs

```diff
@@ -528,6 +528,31 @@ public NumberAnnotation(AnnotationCollection mem)
         public IEnumerable<string> Errors => currentError == null ? Array.Empty<string>() : new[] { currentError };
     }
 
+    class DateTimeAnnotation : IStringValueAnnotation, ICopyStringValueAnnotation
+    {
+        private AnnotationCollection annotation;
+
+        public string Value
+        {
+            get
+            {
+                if (annotation.Get<IObjectValueAnnotation>(from: this).Value is DateTime dt)
+                    return dt.ToString(CultureInfo.CurrentCulture);
+                return "";
+            }
+            set
+            {
+                if (DateTime.TryParse(value, out var dt))
```

Comment:
Here it would be better to do DateTime.Parse, since that would cause a FormatException in case the string is not a valid date time and that turns into a notification for the user.